### DISCUSSION TITLE
Make key.Binding marshable

### DISCUSSION THREAD

**Discussion by antonmedv** [charmbracelet/bubbles#499] [Ideas]
I have a default key binding: https://github.com/antonmedv/fx/blob/master/keymap.go

I want users to be able to define their own bindings. For that I'd like to be able to load key.Bindings from JSON.

**Comment by wI2L** [comment]
@antonmedv Interested in this feature too. Have you implemented it yourself for/in your project about that?

**Reply by antonmedv** [reply]
Nope. Just can’t create config for keybindings right now.

**Comment by activey** [comment]
Loading json should not be part of a core framework, whats makes it so hard to write it yourself?

**Reply by antonmedv** [reply]
>  whats makes it so hard to write it yourself?

It is simple.

But easy marshal/unmarshal key.Binding makes DX superior.
